docs/crystal_for_rubyists/README.md

```diff
@@ -404,6 +404,46 @@ private def method
 end
 ```
 
+### Hash syntax from Ruby to Crystal
+
+Crystal introduces a data type that is not available in Ruby, the NamedTuple.
+
+Typically in Ruby you can define a hash with several syntaxes:
+
+```ruby
+# A valid ruby hash declaration
+{ 
+  key1: "some value",
+  some_key2: "second value"
+}
+
+# This syntax in ruby is shorthand for the hash rocket => syntax
+{
+  :key1 => "some value",
+  :some_key2 => "second value"
```

Comment:
Do we want an example with string keys on both sides or does that make it too complicated?

Reply:
The goal here is just to explain the syntax difference, so I hadn't planned on explaining the differences in how keys are handled.